src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,203 @@
+package duke;
+
+import java.util.ArrayList;
+/**
+ * Represents a task list.
+ */
+public class TaskList {
+    private static ArrayList<Task> list;
+    private static Storage store;
+
+    /**
+     * Creates a task list based on the stored array.
+     * @param store The storage used.
+     */
+    public TaskList(Storage store) throws DukeException{
+        this.store = store;
+        this.list = store.load();
+    }
+
+    /**
+     * Creates an empty task list.
+     */
+    public TaskList() throws DukeException{
+
+        this.list = new ArrayList<Task>(100);
+    }
+    /**
+     * Creates the message when task is added to tasklist.
+     */
+    public static void addTask() throws DukeException{
+        System.out.println("Got it. I've added this task:\n");
+
+        System.out.println(list.get(list.size()-1).ToString());
+
+        System.out.println("Now you have " + list.size() + " tasks in the list.");
+        store.save();
+    }
+
+    /**
+     * Creates an event in the tasklist.
+     * @param str The string representation of the event.
+     */
+    public static void eventCase(String str) throws DukeException {
+        str = str.replace("event", "");
```

Comment:
According to the documentation, you should not reuse formal parameters as local variables inside the method.

```suggestion
        String parsedString = str.replace("event", "");
```